activities/Abecedarium.activity/play.js

```diff
@@ -288,6 +303,34 @@ enyo.kind({
 			this.selected = entry;
 		}
 	},
+
+	showGameFinished: function(){
+		this.cleanBox();
+		Abcd.changeVisibility(this, {home: false, back: false, filter: false, check: false, itemCount: false});
+		Abcd.hideLang();
+		this.$.caseButton.hide();
+		var l10n ={"en":"Game Finished","fr":"Game Finished","es":"Game Finished","pt":"Game Finished","de":"Game Finished"};
```

Comment:
Ooch, that's not at all the right way to do that. See tutorial on localization.

Reply:
Saw something similar in another activity so I went with it as I was not sure how to add localization to enyo activities. Figured it out, It has been fixed now.